swagger.yaml

```diff
@@ -0,0 +1,2651 @@
+      parameters:
+        - description: TODO
+          in: path
+          name: index
+          required: 'true'
+          type: string
+        - description: TODO
+          in: path
+          name: collection
+          required: 'true'
+          type: string
+      produces:
+        - application/json
+      responses:
+        '200':
+          description: ''
+          schema: {}
+  '/:index/:collection/_create':
+    post:
+      description: 'Controller: write. Action: create.'
+      parameters:
+        - description: TODO
+          in: path
+          name: index
+          required: 'true'
+          type: string
+        - description: TODO
+          in: path
+          name: collection
+          required: 'true'
+          type: string
+      produces:
+        - application/json
+      responses:
+        '200':
+          description: ''
+          schema: {}
+  '/:index/:collection/_mapping':
+    put:
+      description: 'Controller: admin. Action: updateMapping.'
+      parameters:
+        - description: TODO
+          in: path
+          name: index
+          required: 'true'
+          type: string
+        - description: TODO
+          in: path
+          name: collection
+          required: 'true'
+          type: string
+      produces:
+        - application/json
+      responses:
+        '200':
+          description: ''
+          schema: {}
+  '/:index/:collection/_query':
+    delete:
+      description: 'Controller: write. Action: deleteByQuery.'
+      parameters:
+        - description: TODO
+          in: path
+          name: index
+          required: 'true'
+          type: string
+        - description: TODO
+          in: path
+          name: collection
+          required: 'true'
+          type: string
+      produces:
+        - application/json
+      responses:
+        '200':
+          description: ''
+          schema: {}
+  '/:index/:collection/_search':
+    post:
+      description: 'Controller: read. Action: search.'
+      parameters:
+        - description: TODO
+          in: path
+          name: index
+          required: 'true'
+          type: string
+        - description: TODO
+          in: path
+          name: collection
+          required: 'true'
+          type: string
+      produces:
+        - application/json
+      responses:
+        '200':
+          description: ''
+          schema: {}
+  '/:index/:collection/_truncate':
+    delete:
+      description: 'Controller: admin. Action: truncateCollection.'
+      parameters:
+        - description: TODO
+          in: path
+          name: index
+          required: 'true'
+          type: string
+        - description: TODO
+          in: path
+          name: collection
+          required: 'true'
+          type: string
+      produces:
+        - application/json
+      responses:
+        '200':
+          description: ''
+          schema: {}
+  '/:index/_autoRefresh':
```

Comment:
missing also "get" method for route '/:index/_autoRefresh' => admin/getAutorefresh